package.json

```diff
@@ -15,7 +15,7 @@
      "peg": "pegjs --allowed-start-rules Definitions,Type --dependency \"./type-structures\" app/types/type-parser.pegjs && mv app/types/type-parser.js app/types/type-parser-inner.js && sed -i \"\" 's/^var \\.\\/type-structures/var types/' app/types/type-parser-inner.js",
```

Comment:
`"pegjs --allowed-start-rules Definitions,Type --dependency \"types:./type-structures\" app/types/type-parser.pegjs && mv app/types/type-parser.js app/types/type-parser-inner.js"`

Is this what you're asking for?

And what do you mean it copies it to the "app/" folder? It looks like the mv command just renames it?